autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -343,6 +343,11 @@ def fit(train_data,
                 Keyword arguments to pass to all models. See the `AG_args_fit` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
                 Identical to specifying `AG_args_fit` parameter for all models in `hyperparameters`.
                 If a key in `AG_args_fit` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+            excluded_model_types : list, default = []
+                Banned subset of model types to avoid training during `fit()`, even if present in `hyperparameters`.
+                Reference `hyperparameters` documentation for the list of model types.
+                Useful when a particular model type such as 'KNN' or 'custom' is not desired but altering the `hyperparameters` dictionary is difficult or time-consuming.
```

Comment:
Since `excluded_model_types` arg will help people who are unfamiliar with the `hyperparameters` documentation/details, I'd remove this sentence and then add some clarification like this to the `hyperparameters` documentation:

"Omitting a model key from `hyperparameters` is equivalent to including this model in `excluded_model_types`."

This way we basically assume the `excluded_model_types` users are not necessarily familiar with `hyperparameters` details, whereas we assume the (more advanced)  `hyperparameters` users are familiar with `excluded_model_types` details.

Reply:
Added